deploy/charts/harvester/crds/harvesterhci.io_virtualmachinerestores.yaml

```diff
@@ -14,8 +14,8 @@ spec:
     listKind: VirtualMachineRestoreList
     plural: virtualmachinerestores
     shortNames:
-    - vmrestore
-    - vmrestores
+    - vm-restore
+    - vm-restores
```

Comment:
kubevirt already uses this one and it is higher prioritized, but since users can append group version at the end of short 
name so let's change it back to `vmrestore` instead thanks.